vpr/src/route/router_delay_profiling.cpp

```diff
@@ -19,7 +19,7 @@ RouterDelayProfiler::RouterDelayProfiler(
           g_vpr_ctx.device().rr_nodes,
           &g_vpr_ctx.device().rr_graph,
           g_vpr_ctx.device().rr_rc_data,
-          g_vpr_ctx.device().rr_switch_inf,
+          g_vpr_ctx.mutable_device().rr_graph_builder.rr_switch_inf(),
```

Comment:
Should not use a mutable object. A profiling function is not allowed to modify any routing resource graph.